examples/main.cpp

```diff
@@ -175,7 +176,10 @@ int run(const char *device_name,
   CALIPER(mgr.start();)
   CALIPER(CALI_MARK_BEGIN("Setup");)
 
-  const bool use_device = std::strcmp(device_name, "cpu") != 0;
+  const bool physics_use_device = ((std::strcmp(device_name, "gpu_cpu") == 0) || std::strcmp(device_name, "gpu_gpu") == 0);
+  const bool ml_use_device = ((std::strcmp(device_name, "gpu_cpu") == 0) || std::strcmp(device_name, "gpu_gpu") == 0);
```

Comment:
I think here we have a bug. <physics_device>_<ml_device>. So, I believe the first condition "gpu_cpu" should be "cpu_gpu".

Reply:
Noted and fixed.